BASE/src/Microsoft.ApplicationInsights/Extensibility/TelemetryConfiguration.cs

```diff
@@ -339,7 +339,9 @@ public string ConnectionString
 
         /// <summary>
         /// Gets an envelope for Azure.Core.TokenCredential which provides an AAD Authenticated token.
+        /// FOR INTERNAL USE ONLY. To set the Credential use <see cref="SetAzureTokenCredential"/>.
         /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
         public CredentialEnvelope CredentialEnvelope { get; private set; }
```

Comment:
I might have missed details for making this property as public. Is there a reason to keep this as public property?

Reply:
needed to change it to public so the QuickPulseTelemetryModule can read it when initialized.